view/frontend/layout/catalog_product_view.xml

```diff
@@ -7,8 +7,13 @@
 -->
 <page xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd">
     <body>
-        <referenceBlock name="product.info.addtocart">
+        <referenceBlock name="product.info.options.wrapper.bottom">
```

Comment:
For configurable product type we can use next xml:
catalog_product_view_type_configurable.xml
For other product types:
catalog_product_view_type_simple.xml
catalog_product_view_type_bundle.xml
catalog_product_view_type_grouped.xml
catalog_product_view_type_virtual.xml
catalog_product_view_type_downloadable.xml

It will be better to avoid possible blocks "duplication" in some specific merchant theme/code.
Also have we tested on all product types ?

Reply:
@garmashdev we also have existing block duplication for the PDP checkout button too 
```
<block class="Bolt\Boltpay\Block\JsProductPage" name="bolt.button" template="Bolt_Boltpay::button_product_page.phtml" before="-" />
```
which is why I added here.

And I think we want to have this on all product types, not only configurable. Certainly we want it for simple + configurable to start.

What would you like us to do?:
1. Create 2 additional xml files: `catalog_product_view_type_simple.xml` and `..._configurable.xml` and add each block there for better scoping, or 
2. Keep the duplicating in the same page as currently, and just test if there are no duplication for all product types. Following the button_product_page.phtml duplication

1. is safer and sounds like what you are recommending. Downside if we will need to add new xml for each product type. I wanted us to leverage `catalog_product_view.xml` which is meant to cover everything.

2. is what we are copying from button_product_page.phtml

Reply:
We can try to left only one reference as global in catalog_product_view.xml 
 ` <block class="Bolt\Boltpay\Block\JsProductPage" name="bolt.frequencySelector" template="Bolt_Boltpay::subscription_freq_selector_product_page.phtml" before="-" />`
and remove there `bolt.frequencySelector.additional` block.
and move `bolt.frequencySelector` block to another block(which is needs for configurable product type) using "move" directive in file `catalog_product_view_type_configurable.xml` as next:
`<move element="bolt.frequencySelector" destination="product.info.options.wrapper.bottom" before="-"/>`

Reply:
Would this apply even for non-configurable products? We want this configuration to apply for all products, not just "configurable" products

Reply:
moving directive will be "true" only for configurable product type, because we are writting it in type_configurable.xml. For all other product types will be true catalog_product_view.xml. And if it needs we can move that block for another types in the same way. I hope it is only one special case for configurable product type. It is correct way, because if merchant created 2 blocks manually on their own magento-theme: "product.info.options.wrapper.bottom" and "product.info.form.content" we will have two the same our blocks.

Reply:
we made the changes per your comments @garmashdev thank you for the recommendation. Please take a look now.